support/gitlab-runners/kubeconfig.yaml

```diff
@@ -1,4 +1,34 @@
 # Copyright 2021-2023 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-To get the KUBECONFIG, it's currently stored either in LastPass or Gitlab CI Variable
+apiVersion: v1
+clusters:
+  - cluster:
+      certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM1ekNDQWMrZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRJeE1EY3lNREl4TkRRME5Gb1hEVE14TURjeE9ESXhORFEwTkZvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTDUxCkdwVW9DKzlnQlNPWENxR3UzOGxKRnh4Qzc2dEdsNEQra3BLWFlqdlNDSVI3alF4ckc3cGh4OGxpQ3NodEdPS3EKTkJraEg3dENpUTdmNm1yQUE4WWhZbkZ1eWRTNDdkUXlXMzlvNWxFZXc1SmQzdm9NTGhRRFZXZGFTSU12TmZpTwpXVVpPOHBhOEtFSkNkeWp6WVcvWmxxK0xscWlZSlRONGI4MEdQcURjUGdLMW5ZVlVHdzdkNUEwN1J4bU9DTCtjCmNwbmJTNm1VOC9UQzBhUUFKK0Rlc094dTM5KzZ0WHR4TFlNM3lsY3NyZjNiMlZwVk9paGZ1SHYvMjNBWTdDamIKTWlPK1FXUTlHWXorWXpvQThwK2xJWFpEUEYyYWlMNThRYUxhSDBmdXhpRk9DZUl6OEQ0VUZSbHRVNk9tZkZwaAp6bmdnMHkxWWFrV0o1eHZhczdjQ0F3RUFBYU5DTUVBd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0hRWURWUjBPQkJZRUZFQm8rQy9pMGhwWGQranExQ2tNSllRL25CQzhNQTBHQ1NxR1NJYjMKRFFFQkN3VUFBNElCQVFCY0JlcFhsa3hMWDFVSjh2R3hPY0tpT2JNZkdsQ0xqTDczZnZsYzRsZkNuWHd1NXVObgpYc3FGWUFMSTdKeFNQTTZTK0ZMMDZTcm16WVNwS3k4NkpLZTRmVVVvcjh1ckpNZ0tqK1d3aE5XdmloWWpUUWZvCjJuQS9vZHJwK0JWcitycVJCREhYN2R3aEhoTmRsZ3orZTJqRlNHdTUvbnZ5NDgrc0VqTUVhamRoWXJhaVozZmUKSTBlNnlNeW81NXc0U0RTTzNMT1UwZFFBOExsbm9PWHVjSUNsaklxY21jbDhtRytVYmVCdDRiemFwemhSWUtxWQp2cytxbkpyK2hNQVkyek1HL3hTWHorUURqMERuc1JhaE44bE9XdmprL3ZEakw1akRhRkk4ZUV0RDlZU1AvOFQ3CmNDYm5XUGhuQ3NVSHQ4VGRjUkRKaVBXUzYyU084b0VXd1VMOAotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
The KUBECONFIG is considered sensitive data and must not be in source control unless it's encrypted. 

It's sensitive even though it does not contain passwords and such because it reveals information about our service and how to connect to it (aws region, cluster name, URIs, CA public key) making it easier to attack.

Reply:
Did not mean to submit this.

Reply:
I will remove this from the commit history.